.github/workflows/deploy.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: 'lts/*'
```

Comment:
that's not very good idea and can break stuff in 6 months, as some package admins are not updating binary dependencies - I've encountering this kind of problems every year or two.
please just use `16` if you want